README.md

```diff
@@ -1,43 +1,57 @@
 sbp_mcafee Cookbook
 ===================
-This cookbook will install the McAfee-client on Windows-systems
 
+[![Build Status](https://travis-ci.org/sbp-cookbooks/sbp_mcafee.svg?branch=master)](https://travis-ci.org/sbp-cookbooks/sbp_mcafee) [![Cookbook Version](https://img.shields.io/cookbook/v/sbp_mcafee.svg)](https://supermarket.chef.io/cookbooks/sbp_mcafee)
+
+This cookbook will install the McAfee-client on Windows-systems
 
 Requirements
 ------------
-The cookbook depends on the windows cookbook, the client should be managed from an EPO server
 
+The cookbook depends on the windows cookbook, the client should be managed from an EPO server
 
 Attributes
 ----------
+
+```ruby
 default['mcafee']['agent']['package_name']      = 'McAfee Agent'
 default['mcafee']['agent']['url']               = ''
 default['mcafee']['virusscan']['package_name']  = 'McAfee VirusScan Enterprise'
 default['mcafee']['virusscan']['url']           = ''
+```
 
 For the agent you'll need the FramePkg.exe and for the virusscan the av.zip.
 Both files should be placed on your local repo and defined in the attributes.
 
+The cookbook now also supports installation of McAfee Endpoint Security.
+In order to do so ovrewrite the virusscan attributes like this
+
+```ruby
+overwrite['mcafee']['virusscan']['package_name']      = 'McAfee Endpoint Security Platform'
+overwrite['mcafee']['virusscan']['options']           = 'ADDLOCAL="tp" /qn /nocontentupdate'
+overwrite['mcafee']['virusscan']['installer_exe']     = 'setupEP.exe'
```

Comment:
There is no need to override them. Just setting them is enough.

Additionally it's `override`, not `overwrite` :wink:

Reply:
fixed